src/scss/includes/components/closeButton.scss

```diff
@@ -0,0 +1,20 @@
+.closeButton {
+  height: 35px;
+  width: 35px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 22px;
+  cursor: pointer;
+  margin-top: -9px;
```

Comment:
why a negative margin is needed here ? Shouldn't setting w/h dimensions be enough ?

Reply:
That's what I explained here https://github.com/QwantResearch/erdapfel/pull/815#discussion_r498821104 and referred to also there https://github.com/QwantResearch/erdapfel/pull/815#discussion_r498833356.
![Capture d’écran de 2020-10-05 13-52-59](https://user-images.githubusercontent.com/243653/95076688-89584000-0712-11eb-96e9-92bf01cd06dd.png)

We need to 'cheat' so that the button content (the cross) is aligned to the content, but the actual active area should be larger than that. I think it's best if this is the responbility of the button itself, as it's dependant of its size, and part of what justified to make a separate component.
But I understand if it bothers you that it's directly in the class. Maybe we can put it a variant like `.closeButton--topRight' and map a prop to it?

Reply:
I missed your previous comment has it had been marked as resolved, sorry.
The button would be reusable anywhere if we can pass props to specify margins :+1:

Reply:
Just to be sure we're on the same line: I don't want to pass to margin values themselves, as they depend on the button style itself. I want to pass props which say that the button is used on the right/top of a container, so it can be shifted.
Like:
`<CloseButton onClose={foo} position="topRight" />`
and for CSS:
```scss
.closeButton {
  // general style

  &--topRight {
    margin-top: -9px;
    margin-right: -9px;
  }
}
```

Reply:
yep it's ok :+1: